packages/api/core/test/fast/require-search_spec.ts

```diff
@@ -1,22 +1,29 @@
 import { expect } from 'chai';
 
 import findConfig from '../../src/util/forge-config';
-import requireSearch from '../../src/util/require-search';
+import importSearch from '../../src/util/import-search';
 
-describe('require-search', () => {
-  it('should resolve null if no file exists', () => {
-    const resolved = requireSearch(__dirname, ['../../src/util/wizard-secrets']);
+describe('import-search', () => {
+  it('should resolve null if no file exists', async () => {
+    const resolved = await importSearch(__dirname, ['../../src/util/wizard-secrets']);
     expect(resolved).to.equal(null);
   });
 
-  it('should resolve a file if it exists', () => {
-    const resolved = requireSearch(__dirname, ['../../src/util/forge-config']);
+  it('should resolve a file if it exists', async () => {
+    const resolved = await importSearch(__dirname, ['../../src/util/forge-config']);
     expect(resolved).to.equal(findConfig);
   });
 
-  it('should throw if file exists but fails to load', () => {
-    expect(() => {
-      requireSearch(__dirname, ['../fixture/require-search/throw-error']);
-    }).to.throw('test');
+  it('should throw if file exists but fails to load', async () => {
+    let test: () => unknown;
+    try {
+      const ret = await importSearch(__dirname, ['../fixture/require-search/throw-error']);
+      test = () => ret;
+    } catch (error) {
+      test = () => {
+        throw error;
+      };
+    }
+    expect(test).to.throw('test');
```

Comment:
I believe we can use the `to.be.rejected` assertion from `chai-as-promised` to simplify the boilerplate for testing promise rejections:

docs: https://www.chaijs.com/plugins/chai-as-promised/

example: https://github.com/electron/forge/blob/84eb77568a89330999dafb531a9ed7e003870165/packages/plugin/webpack/test/WebpackConfig_spec.ts#L196

Reply:
> I believe we can use the `to.be.rejected` assertion from `chai-as-promised` to simplify the boilerplate for testing promise rejections: (...)

I've applied your suggestion in code, I think this even fits more the overall style of the other tests right now in this file, not only being much more simpler. I've also added you as co-author to this single commit, since it was your suggestion that made me to improve it.